jdi-light-bootstrap/src/main/java/com/epam/jdi/light/ui/bootstrap/elements/composite/Form.java

```diff
@@ -1,6 +1,168 @@
 package com.epam.jdi.light.ui.bootstrap.elements.composite;
 
-import com.epam.jdi.light.elements.composite.Section;
+import com.epam.jdi.light.common.Exceptions;
+import com.epam.jdi.light.common.JDIAction;
+import com.epam.jdi.light.elements.base.UIBaseElement;
+import com.epam.jdi.light.elements.common.UIElement;
+import com.epam.jdi.light.elements.complex.WebList;
+import com.epam.jdi.tools.ReflectionUtils;
+import org.openqa.selenium.WebElement;
+
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static com.epam.jdi.light.elements.pageobjects.annotations.WebAnnotationsUtil.getElementName;
+
+public class Form<T> extends com.epam.jdi.light.elements.composite.Form<T> {
```

Comment:
Is it good practice?

Reply:
The fully-qualified class name is the only way in case we have two ore more classes with same names.